src/preamble.js

```diff
@@ -1764,6 +1764,7 @@ var Math_imul = Math.imul;
 var Math_fround = Math.fround;
 var Math_min = Math.min;
 var Math_clz32 = Math.clz32;
+var Math_trunc = Math.trunc;
```

Comment:
Ok, must be somewhere that depends on it, that we can probably remove. But not a problem for this pr anyhow.